AmplifyPlugins/API/AWSAPICategoryPluginFunctionalTests/GraphQLModelBased/GraphQLModelBasedTests.swift

```diff
@@ -294,6 +295,54 @@ class GraphQLModelBasedTests: XCTestCase {
         wait(for: [completeInvoked], timeout: TestCommonConstants.networkTimeout)
     }
 
+    func testUpdatePostWithInvalidCondition() {
+        let uuid = UUID().uuidString
+        let testMethodName = String("\(#function)".dropLast(2))
+        let title = testMethodName + "Title"
+        guard let createPost = createPost(id: uuid, title: title) else {
+            XCTFail("Failed to ensure at least one Post to be retrieved on the listQuery")
+            return
+        }
+        let conditionTitle = "IdontCareUpdated"
```

Comment:
the test is "testUpdatePostWithInvalidCondition", perfer the naming of the variable match closely ie.
```
let invalidCondition = post.title == "invalidTitle"
```
also there's some verbosity with setting up the test with one-time use variables, but in this case, creating the condition is part of setting the test up, and i don't think we need an extra variable like "let conditionTitle"

Reply:
true. Changed